docs/sso/README.md

```diff
@@ -0,0 +1,14 @@
+# Single Sign On
+
+
+## Frontend process
+
+The user sign in using his GitHub account, and we receive the GitHub token with his information as email, name, username, image, etc., after we send the email, username, GitHub token, and a secret to identify the application which is sending the request is our frontend nor other one trying to sign in using our CKAN, after that receiving the response we use the API token to communicate with CKAN identifying him.
+
+## Backend process 
+
+In CKAN we receive the data sent by the frontend and validate first the secret to check if the stored secret in CKAN matches the received one, after that we verify the token to check if it's a token generated from GitHub.
```

Comment:
It sounds weird to use "we" in this context. I mean, CKAN is not "we".

I also suggest rewriting it into a numbered list.